CONTRIBUTING.md

```diff
@@ -5,33 +5,44 @@ meeting](https://github.com/open-telemetry/community#special-interest-groups).
-them, stability in the behavior is important, be it runtime or configuration. They are more numerous and harder to get
-in touch with, making our changes to the collector more disruptive to them than to other audiences. As a general rule,
-whenever you are developing OpenTelemetry Collector components (extensions, receivers, processors, exporters), you
-should have end-users' interests in mind. Similarly, changes to code within packages like `config` will have an impact
-on this audience. Make sure to cause minimal disruption when doing changes here.
-
-### Collector distributions
-
-In this capacity, the opentelemetry-collector repository's public Go types, functions, and interfaces act as an API for
-other projects. In addition to the end-user aspect mentioned above, this audience also cares about API compatibility,
-making them susceptible to our refactorings, even though such changes wouldn't cause any impact to end-users. See the
-"Breaking changes" in this document for more information on how to perform changes affecting this audience.
-
-This audience might use tools like the
-[opentelemetry-collector-builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder) as
-part of their delivery pipeline. Be mindful that changes there might cause disruption to this audience.
+[opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository, as
+well as distributions created using the [OpenTelemetry Collector
+Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder). To them, stability in the
+behavior is important, be it runtime or configuration. They are more numerous and harder to get in touch with, making
+our changes to the Collector more disruptive to them than to other audiences. As a general rule, whenever you are
+developing OpenTelemetry Collector components (extensions, receivers, processors, exporters, connectors), you should
+have end-users' interests in mind. Similarly, changes to code within packages like `config` will have an impact on this
+audience. Make sure to cause minimal disruption when doing changes here.
+
+### Component developers
+
+Component developers create new extensions, receivers, processors, exporters and connectors to be used with the
+OpenTelemetry Collector. They are the primary audience for the opentelemetry-collector repository's public Go API. A
+significant part of them will contribute to opentelemetry-collector-contrib. In addition to the end-user aspect
+mentioned above, this audience also cares about Go API compatibility of Go modules such as the ones in the `pdata`,
+`component`, `consumer`, `confmap`, `exporterhelper` and other modules, even though such changes wouldn't cause any
+impact to end-users. See the "Breaking changes" in this document for more information on how to perform changes
+affecting this audience.
+
+### Collector library users
```

Comment:
Let's mention that these are our most advanced users and are the most equipped to handle disruptive changes.